### PR TITLE
Rust 1.56.1 release blog post

### PR DESCRIPTION
**Pull request by pietroalbini**
Announcement for the 1.56.1 release.

cc @rust-lang/security @rust-lang/release

### CODE REVIEW

**Comment by pietroalbini** [comment]
Note that it will likely take a while for the release to be out, I'm just opening the PR early to get it approved :slightly_smiling_face:

File: posts/2021-11-01-Rust-1.56.1.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Announcing Rust 1.56.1"
+author: The Rust Release Tea
```
Comment (badboy) [inline posts/2021-11-01-Rust-1.56.1.md:4]:
```suggestion
author: The Rust Release Team
```

**Review by badboy — APPROVED** [review]
With the author name fixed, this is good to go.

File: posts/2021-11-01-Rust-1.56.1.md (hunk 1 above)
Comment (pietroalbini) [reply posts/2021-11-01-Rust-1.56.1.md]: This was actually supposed to be the security response wg :sweat_smile: Dunno what happened, fixed.

**Review by badboy — APPROVED** [review]

**Pull request merged**